src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private final User secondUser = new User();
+    private final User thirdUser = new User();
+    private User defaultUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        String loginDefault = "Kate";
+        defaultUser = new User();
+        defaultUser.setLogin(loginDefault);
+        defaultUser.setAge(RegistrationServiceImpl.MIN_AGE_VALUE);
+        String goodPassword = "123456";
+        defaultUser.setPassword(goodPassword);
+        String loginBruce = "Bruce";
+        int validUser2Age = 100;
+        String reverseGoodPassword = "654321";
+        secondUser.setLogin(loginBruce);
+        secondUser.setAge(validUser2Age);
+        secondUser.setPassword(reverseGoodPassword);
+    }
```

Comment:
no need to create variables for values that you use in the next line. It has to be clear by method name what is checked in the method. Same with other methods(tests)